config/crd/bases/networking.kubeslice.io_slices.yaml

```diff
@@ -6,20 +6,20 @@ metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.7.0
   creationTimestamp: null
-  name: slice.networking.kubeslice.io
+  name: slices.networking.kubeslice.io
```

Comment:
I dont think we should change the API name. You simply have to change the plural field to "slices". No other change is needed.